tooling/commands/ci.js

```diff
@@ -45,8 +45,8 @@ module.exports = {
   },
   handler: wrapHandler(async (argv) => {
     let packages = argv.integration ?
-      await updated({dependents: true, npm: true}) :
-      await updated({npm: true});
+      await updated({dependents: true, npm: !!process.env.CI}) :
```

Comment:
This is supposed to run on ci, so can we leave this as a manual `true`?

Reply:
This file actually doesn't run anymore since we moved to a bash based loop